eva/optimizer/statement_to_opr_convertor.py

```diff
@@ -222,8 +222,11 @@ def visit_load_data(self, statement: LoadDataStatement):
         if table_metainfo is None:
             # Create a new metadata object
             table_metainfo = create_video_metadata(table_ref.table.table_name)
-
-        load_data_opr = LogicalLoadData(table_metainfo, statement.path)
+            
+        load_data_opr = LogicalLoadData(table_metainfo, 
+                                        statement.path, 
+                                        statement.column_list,
```

Comment:
Get the column_metadata object using  column_list.
`column_metadata_list = create_column_metadata(statement.column_list)`

Reply:
Ohh sure sure